wagl/modtran.py

```diff
@@ -158,84 +166,123 @@ def format_tp5(acquisitions, ancillary_group, satellite_solar_group,
     iso_time = acquisitions[0].acquisition_datetime.isoformat()
     group.attrs['acquisition-datetime'] = iso_time
 
-    tp5_data = {}
-
-    # setup the tp5 files required by MODTRAN
+    json_data = {}
+    # setup the json files required by MODTRAN
     if workflow == Workflow.STANDARD or workflow == Workflow.NBAR:
         acqs = [a for a in acquisitions if a.band_type == BandType.REFLECTIVE]
 
         for p in range(npoints):
+
             for alb in Workflow.NBAR.albedos:
-                input_data = {'water': water_vapour,
-                              'ozone': ozone,
-                              'filter_function': acqs[0].spectral_filter_file,
-                              'visibility': -aerosol,
+                name = 'POINT-%i-ALBEDO-%s' %(p,str(alb.value))
```

Comment:
Replace this with _wagl/constants.POINT_ALBEDO_FMT_.  That way the name can be changed in one place.
You can use it using:
```python
name = POINT_ALBEDO_FMT.format(p=p, a=alb.value)
```

Reply:
this had been addressed now